src/main/java/org/folio/service/impl/CheckOutServiceImpl.java

```diff
@@ -50,15 +54,33 @@ public CheckOutResponse checkOut(CheckOutRequest checkOutRequest) {
   }
 
   private LoanPolicy retrieveLoanPolicy(CheckOutRequest checkOutRequest) {
-    var checkOutDryRunResponse = checkOutClient.checkOutDryRun(checkOutDryRunRequestMapper
-      .mapCheckOutRequestToCheckOutDryRunRequest(checkOutRequest));
+    log.info("retrieveLoanPolicy:: checkOutRequest: {}", checkOutRequest);
+    var checkOutDryRunResponse = checkOutClient.checkOutDryRun(
+      checkOutDryRunRequestMapper.mapCheckOutRequestToCheckOutDryRunRequest(checkOutRequest),
+      extractHeaders());
```

Comment:
Why do we need to copy these headers? Something to do with the desired permissions?